packages/flutter/lib/src/rendering/view.dart

```diff
@@ -184,6 +204,35 @@ class RenderView extends RenderObject with RenderObjectWithChildMixin<RenderBox>
     }
   }
 
+  void _updateSystemChrome() {
+    if (!automaticSystemUiAdjustment)
```

Comment:
i'd put the if at the call site. it's weird for a method called "updateFoo" to not unconditionally update Foo when called.

Reply:
Fixed